main/routing.py

```diff
@@ -3,6 +3,6 @@
 from main.consumer import Consumer
 
 websocket_urlpatterns = [
-    re_path(r'ws/socket/(?P<address>[\w+:]+)/$', Consumer.as_asgi()),
+    re_path(r'api/subscription/(?P<address>[\w+:]+)/$', Consumer.as_asgi()),
```

Comment:
@monmon29 Let's make this `ws/watch/bch | slp/<address>/[token_id]`. Strictly speaking, this is not subscription step anymore, it's more of watching for transactions in an address that has been subscribed earlier through the API.

Reply:
Make sense. With that, here's a commit for that. -->  2ad91d940698b60059a4cea5f78f8c0f5d8fece2 .